frontend/packages/insights-plugin/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@console/insights-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Plugin for Insights operator",
+  "private": true,
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "yarn --cwd ../.. run test packages/insights-plugin"
+  },
+  "dependencies": {
+    "@console/ceph-storage-plugin": "0.0.0-fixed",
+    "@console/container-security": "0.0.0-fixed",
+    "@console/dev-console": "0.0.0-fixed",
+    "@console/internal": "0.0.0-fixed",
+    "@console/knative-plugin": "0.0.0-fixed",
+    "@console/kubevirt-plugin": "0.0.0-fixed",
+    "@console/local-storage-operator-plugin": "0.0.0-fixed",
+    "@console/metal3-plugin": "0.0.0-fixed",
+    "@console/network-attachment-definition-plugin": "0.0.0-fixed",
+    "@console/noobaa-storage-plugin": "0.0.0-fixed",
+    "@console/operator-lifecycle-manager": "0.0.0-fixed",
```

Comment:
I assume you don't really depend on all of these? You should only need to specify dependencies specific to your plugin here.